build-tsunami-package

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+################################################################################
+#
+# Helper script to build a deb package for MXNet
+#
+# This script provides some custom handling of git tags to allow us to tag
+# package releases with well-crafted Debian package versions, e.g.:
+#
+#     1.0.0-0tsunami0
```

Comment:
I note that this probably isn't a valid `debian_revision` even by the standards being used: it should be `0tsunami1` ;-)